src/js/utils/__tests__/ErrorMessageUtil-test.js

```diff
@@ -0,0 +1,185 @@
+jest.dontMock('../ErrorMessageUtil');
+
+const ErrorMessageUtil = require('../ErrorMessageUtil');
+
+describe('ErrorMessageUtil', function () {
```

Comment:
The function is never touching `path` so I didn't thought it would make sense. But you might have a point for actually ensuring that it's not touching it 😄